README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
```

Comment:
```suggestion
- Development times are significantly faster than other cross-platform frameworks thanks to stateful hot-reloading and excellent virtual device support.
```
_updating_ the application in "real time" is not strictly accurate. 
if anything the App can be reloaded on a device connected to the IDE in "soft real time". 
see: https://stackoverflow.com/questions/17308956/differences-between-hard-real-time-soft-real-time

Reply:
Truth is not in real time but "Hot reload" allows it to be quite fast.